packages/sdk/src/types/plugins/ModuleFederationPlugin.ts

```diff
@@ -216,6 +220,7 @@ export interface ModuleFederationPluginOptions {
 
   dev?: boolean | PluginDevOptions;
   dts?: boolean | PluginDtsOptions;
+  dataPrefetch?: DataPrefetch;
```

Comment:
The schema should add the field as well (packages/enhanced/src/schemas/container/ModuleFederationPlugin.ts)